src/borrower/feed/test/navfeed.t.sol

```diff
@@ -61,7 +61,7 @@ contract NAVTest is DSTest, Math {
         bytes32 nftID = feed.nftID(mockNFTRegistry, tokenId);
         feed.update(nftID, nftValue, risk);
         shelf.setReturn("shelf",mockNFTRegistry, tokenId);
-        pile.setReturn("debt_loan", 0);
+        pile.incDebt(tokenId, 0);
```

Comment:
Okay, here you are using the real pile instead of a mocked version. Correct?

Reply:
Is there any specific reason?

Reply:
I am using the mock. The mock also has the incDebt function to keep track of multiple loan debts